sdk/containerregistry/azure-containerregistry/azure/containerregistry/_models.py

```diff
@@ -127,58 +146,101 @@ def _to_generated(self) -> ManifestWriteableProperties:
 
     @property
     def architecture(self) -> ArtifactArchitecture:
+        """CPU Architecture of an artifact.
+        :rtype: ~azure.containerregistry.ArtifactArchitecture
```

Comment:
```suggestion
        :rtype: ~azure.containerregistry.ArtifactArchitecture or str
```

Reply:
More than the docstring, change the type line 148 as well

Reply:
Updated both docstring and return type.